inspector/src/test/resources/twitter4j.Logger.json

```diff
@@ -0,0 +1,175 @@
+{
+  "jsonType": "com.mediadriver.atlas.java.v2.JavaClass",
```

Comment:
I know these are only examples, but could we use io.synsesis or io.atlasmap or something instead of com.mediadriver?